WIP: Перевод названий и описаний оружия

Ohar opened the pull request:
## Changelog
:cl:
tweak: Переведены названия и описания шахтёрских КА, их модификаций, и ряда лазеров и тазеров
/:cl:

Ohar committed:
еревод фраз торговых автоматов

Ohar committed:
Фразы торговых автоматов: мелкие правки

Ohar committed:
Merge branch 'ss220-space:master220' into master220

Ohar committed:
Перевод названий и описаний оружия (не всего)

Ohar committed:
Merge branch 'ss220-space:master220' into master220

Ohar committed:
Merge branch 'master220' into weapon

TrashDoxx commented on code/modules/projectiles/guns/energy/kinetic_accelerator.dm:
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/energy/kinetic_accelerator
-	name = "proto-kinetic accelerator"
-	desc = "A self recharging, ranged mining tool that does increased damage in low pressure. Capable of holding up to six slots worth of mod kits."
+	name = "Протокинетический акселератор"
```
```suggestion
	name = "Протокинетический акселератор"
```

TrashDoxx commented on code/modules/projectiles/guns/energy/laser.dm:
```diff
@@ -11,24 +11,24 @@
-	name = "practice laser gun"
-	desc = "A modified version of the basic laser gun, this one fires less concentrated energy bolts designed for target practice."
+	name = "Тренировочный лазерный карабин"
+	desc = "Модифицированная версия базового лазерного карабина. Стреляет менее концентрированными лучами и предназначено для тренировок."
```
```suggestion
	desc = "Модифицированная версия базового лазерного карабина. Стреляет менее концентрированными лучами и предназначена для тренировок."
```

TrashDoxx commented on code/modules/projectiles/guns/energy/laser.dm:
```diff
@@ -11,24 +11,24 @@
 
 /obj/item/gun/energy/laser/retro
-	name ="retro laser gun"
+	name ="Ретро лазерный карабин"
```
```suggestion
	name ="Устаревший лазерный карабин"
```

TrashDoxx commented on code/modules/projectiles/guns/energy/laser.dm:
```diff
@@ -11,24 +11,24 @@
+	name ="Ретро лазерный карабин"
 	icon_state = "retro"
-	desc = "An older model of the basic lasergun, no longer used by Nanotrasen's private security or military forces. Nevertheless, it is still quite deadly and easy to maintain, making it a favorite amongst pirates and other outlaws."
+	desc = "Устаревшая модель базового лазерного карабин, больше не используемая охранными и военными силами NanoTrasen. Тем не менее, он всё же обладает приличной убойной силой, что, вместе с простотой обслуживания, делает ретро лазерный карабин любимым оружием пиратов и прочих преступников."
```
```suggestion
	desc = "Устаревшая модель базового лазерного карабина, больше не используемая охранными и военными силами NanoTrasen. Тем не менее, он всё же обладает приличной убойной силой, что, вместе с простотой обслуживания, делает эту модель любимым оружием пиратов и прочих преступников."
```

TrashDoxx commented on code/modules/projectiles/guns/energy/kinetic_accelerator.dm:
```diff
@@ -531,8 +531,8 @@
-	name = "decrease pressure penalty"
-	desc = "A syndicate modification kit that increases the damage a kinetic accelerator does in high pressure environments."
+	name = "Улучшение эффективности в атмосфере"
+	desc = "Синдикатовская модификация, увеличивающая урон от выстрелов кинетического акселератора в средах с высоким давлением."
```
```suggestion
	desc = "Модификация, разработанная Синдикатом для увеличения урона от выстрелов кинетического акселератора в среде с высоким давлением."
```

TrashDoxx replied:
```suggestion
	desc = "Модификация, разработанная Синдикатом для увеличения урона от выстрелов кинетического акселератора в среде с высоким давлением."
```

TrashDoxx commented on code/modules/projectiles/guns/energy/nuclear.dm:
```diff
@@ -22,8 +22,8 @@
 /obj/item/gun/energy/gun/mini
-	name = "miniature energy gun"
-	desc = "A small, pistol-sized energy gun with a built-in flashlight. It has two settings: disable and kill."
+	name = "Миниатюрный энергопистолет"
+	desc = "Небольшой энергопистолет со встроенным фонариком. У него два режима: оглушающий и летальный."
```
```suggestion
	name = "Миниатюрный энергетический пистолет"
	desc = "Небольшой энергетический пистолет со встроенным фонариком. У него два режима: оглушающий и летальный."
```

TrashDoxx commented on code/modules/projectiles/guns/energy/nuclear.dm:
```diff
@@ -43,8 +43,8 @@
 /obj/item/gun/energy/gun/hos
-	name = "\improper X-01 MultiPhase Energy Gun"
-	desc = "This is an expensive, modern recreation of an antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time."
+	name = "\improper Многофазный энергопистолет Икс-01"
+	desc = "Это дорогая, современная версия антикварного лазерного пистолета. У этого оружия есть несколько уникальных режимов ведения огня, но нет возможности самостоятельно перезаряжаться с течением времени."
```
```suggestion
	name = "\improper Многофазный энергетический пистолет X-01"
	desc = "Это дорогая, современная версия старинного лазерного пистолета. У этого оружия есть несколько уникальных режимов ведения огня, но нет возможности самостоятельно перезаряжаться с течением времени."
```

Ohar committed:
Update code/modules/projectiles/guns/energy/laser.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Update code/modules/projectiles/guns/energy/laser.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Update code/modules/projectiles/guns/energy/laser.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Update code/modules/projectiles/guns/energy/nuclear.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Update code/modules/projectiles/guns/energy/nuclear.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Update code/modules/projectiles/guns/energy/kinetic_accelerator.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Ohar committed:
Акселератор → Ускоритель
https://discord.com/channels/617003227182792704/677928179536625697/951470639976431707

botPurple commented on code/modules/projectiles/guns/energy/kinetic_accelerator.dm:
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/energy/kinetic_accelerator
-	name = "proto-kinetic accelerator"
-	desc = "A self recharging, ranged mining tool that does increased damage in low pressure. Capable of holding up to six slots worth of mod kits."
+	name = "Протокинетический ускоритель"
```
Требует обновления вики

botPurple commented on code/modules/projectiles/guns/energy/kinetic_accelerator.dm:
```diff
@@ -459,8 +459,8 @@
 /obj/item/borg/upgrade/modkit/lifesteal
-	name = "lifesteal crystal"
-	desc = "Causes kinetic accelerator shots to slightly heal the firer on striking a living target."
+	name = "Кристалл кражи жизни"
```
"кражи" на "похищения"

botPurple commented on code/modules/projectiles/guns/energy/laser.dm:
```diff
@@ -11,24 +11,24 @@
 
 /obj/item/gun/energy/laser/captain
-	name = "antique laser gun"
+	name = "Старинный лазерный пистолет"
```
Не понимаю чем тебе не понравилось слово "Антикварный". Перевод потерял смысл английской версии из-за этого

botPurple reviewed (changes requested):
Далеко не всё лазерное оружие переведено. будет вызывать осуждение и кринж, когда будет **You fire Лазерный карабин!** или **That's Лазерный карабин. It is a normal-sized item.** или **Флор Фомичёв fires Лазерный карабин!**
